remoulade/api/main.py

```diff
@@ -121,3 +168,20 @@ def http_exception(e):
 @app.errorhandler(ValidationError)
 def validation_error(e):
     return {"error": e.normalized_messages()}, 400
+
+
+api_spec.register(cancel_message)
+api_spec.register(requeue_message)
+api_spec.register(get_results)
+api_spec.register(enqueue_message)
+api_spec.register(get_actors)
+api_spec.register(get_options)
+api_spec.register(get_states, blueprint="messages")
```

Comment:
it's probably better to export a `routes` variable for each file and loop on it, it will reduce the risk of forgetting to add a new route